src/components/SettingsBarContext.tsx

```diff
@@ -0,0 +1,53 @@
+import { createContext, ReactNode, useState } from 'react';
+import { FeatureStatus } from '../models';
```

Comment:
There is no such exported value in models.
This mean it will brake app once merged

Reply:
I'm waiting for the other my approves and then I will get this essential  values

Reply:
i waited 😁